packages/cli/package.json

```diff
@@ -8,7 +8,6 @@
     "featurevisor": "bin.js"
   },
   "scripts": {
-    "lint": "echo 'not linting in this package yet'",
```

Comment:
worth keeping linting at package level?

from root, we could use lerna to run `link` script for all packages then.